core/src/main/java/org/infinispan/configuration/parsing/CacheParser.java

```diff
@@ -1412,22 +1412,22 @@ private void parseClusterLoader(ConfigurationReader reader, ConfigurationBuilder
    }
 
    protected void parseFileStore(ConfigurationReader reader, ConfigurationBuilderHolder holder) {
-      SoftIndexFileStoreConfigurationBuilder sifsBuilder = null;
+      SoftIndexFileStoreConfigurationBuilder fileStoreBuilder = null;
 
       PersistenceConfigurationBuilder persistence = holder.getCurrentConfigurationBuilder().persistence();
       AbstractStoreConfigurationBuilder<?, ?> actualStoreConfig;
       int majorSchema = reader.getSchema().getMajor();
-      boolean isVersion13 = false;
+      boolean fileIsSIFS = false;
```

Comment:
I think the old name was better, because it's not true in version 14 even though a file-store element still represents a SIFS.

Reply:
I guess the question is if we should allow migration of old SFS to SIFS in Infinispan 14 as well. Currently it only does the migration if the xml schema is 13. But if we are now deprecating data in 14 instead of removing, we may have to also allow the migration too.

Reply:
I've changed the name to `legacyFileStore`.

Regarding the actual configuration: I think it would be better to have the single `path` attribute stay as a way to set both `data` and `index` paths in one fell swoop.

Reply:
@tristantarrant would you want to remove the deprecation message/XSD documentation, then?

Reply:
I have removed the warning message and the deprecation from the xsd,